scripts/generate-docs.sh

```diff
@@ -12,14 +12,22 @@ if [ $(git status --porcelain | wc -l) != "0" ]; then
   exit 1
 fi
 
-echo "Switching to last release..."
-VERSION=$(node scripts/get-version.js)
 EXEC_BRANCH=$(git rev-parse --abbrev-ref HEAD)
+if [ "$#" -eq 0 ]; then
```

Comment:
Because reading bash is hard, can we add a comment here? Maybe:

```
# If a commit ref is passed as a command line option, use that. Otherwise, default to the
# latest tagged version.
```